utils/docker/Dockerfile.ubuntu.20.04

```diff
@@ -23,6 +23,7 @@ RUN apt-get -y update && \
         build-essential \
         clang \
         cmake \
+	curl \
```

Comment:
nit: alignment

Reply:
Yup, i noticed, problem is i used to use tab to align, but it looks it used space originally, just wondering what is coding styles, or docker file this is expected.